web/src/message_view_header.ts

```diff
@@ -170,6 +170,15 @@ export function initialize(): void {
 export function render_title_area(): void {
     const filter = narrow_state.filter();
     build_message_view_header(filter);
+    $(".narrow_description").on("mouseenter", function () {
+        $(this).toggleClass("narrow_description_extended");
+        $(".top-navbar-container").css("overflow", "visible");
```

Comment:
It shouldn't be necessary to both toggle the class and manually call `.css()`, too. Far more preferable to only toggle the class here, and keep the CSS you need in the message view header file.